model/metrics/container.yaml

```diff
@@ -14,6 +14,20 @@ groups:
         brief: "The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
         requirement_level: opt_in
 
+  - id: metric.container.cpu.usage
+    type: metric
+    metric_name: container.cpu.usage
+    stability: experimental
+    brief: "CPU usage"
+    note: >
+      CPU usage of the specific container on all available CPU cores, averaged over the sample window
+    instrument: gauge
+    unit: "{cpu}"
```

Comment:
is this right? should it be time? ticks? percentage?

Reply:
Based on the implementation from [kubeletstats](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.102.0/receiver/kubeletstatsreceiver/documentation.md#containercpuusage) it is `{cpu}`, see https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/25901#issuecomment-1885979641 for more context.

Reply:
In all other utilization metrics we use `1` as a unit. Having `{cpu}` there creates an impression that we measure number of CPUs.

Reply:
But this is not a utilization metric, this is usage metric.
Correct me if I'm wrong here but the intention is to measure the number of CPUs used by the container over a reporting interval. That means essentially that if a container uses 2 complete CPUs constantly during the internal, the metric would be `2`. 
Then if we divide this with the number of Node's cores for example (ie `4` cores) it would give us `2/4=0.5` utilization.
In `kubeletstats` we have other metrics specifically for reporting the utilization against the container's limits and/or the Node's capacity.

cc @TylerHelmuth @dmitryax @povilasv who were involved in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/25901.

Reply:
Correct. We use `utilization` when there is a limit available and count it as a percentage [0,1]. Aggregation of those metrics would be `avg` which result in the same units `1`.

Here, we have an average usage of CPUs. If we had `cpu.logical_number` attribute from docker, the aggregation method would be `sum`, which gives us the same `{cpu}` unit.